common/src/test/java/com/google/tsunami/common/command/CommandExecutorTest.java

```diff
@@ -83,4 +83,16 @@ public void getError_withMultipleGetOutputCalls_returnsExpect()
     assertThat(executor.getError()).isEqualTo("1\n");
     assertThat(executor.getError()).isEqualTo("1\n");
   }
+
+  @Test
+  public void execute_writesInputIfSet()
```

Comment:
Please use the format functionUnderTest_condition_expectedResult
e.g. something like execute_whenInputIsSet_writesInputToCommand